cf-agent/cf_sql.c

```diff
@@ -319,7 +319,7 @@ static void CfDeletePostgresqlQuery(ARG_UNUSED CfdbConn *cfdb)
 
 /*****************************************************************************/
 
-int CfConnectDB(CfdbConn *cfdb, DatabaseType dbtype, char *remotehost, char *dbuser, char *passwd, char *db)
+void CfConnectDB(CfdbConn *cfdb, DatabaseType dbtype, char *remotehost, char *dbuser, char *passwd, char *db)
```

Comment:
Maybe this function should return failure if any of the `CfConnect...DB` functions fail? Just putting the result in `cfdb->data` seems like it might create problems.

That might be a task for another PR however.

Reply:
@karlhto Success / failure is communicated and checked through `cfdb->connected`. Not the way I would do it, but I'm not going to refactor all of these small things. Like you said, it can be done in another PR.

Reply:
Sounds good. :)